mitype/app.py

```diff
@@ -404,3 +411,27 @@ def size_error():
         sys.stdout.write("Window too small to print given text")
         curses.endwin()
         sys.exit(4)
+
+    def save_history(self):
+        # Saving stats in file
+
+        history_file = 'mitype_history.csv'
```

Comment:
Keep the file name `.mitype_history.csv`

Reply:
Ohh I then it would be a hidden file in Linux and not hidden in Windows

Reply:
I previously made it hidden in both OS, but you decided that for deleting history, we should keep it unhidden.

Reply:
Sorry for the confusion. The inital motive wasn't to hide the file as such, just wanted a dotfile as it is a common practice.